### PR TITLE
QDS ADS integration

### PR DESCRIPTION
**Pull request by stuti149**
This PR:

Adds UI to ask customer to collect Ad-hoc queries using QDS and based on selection send boolean parameter to service to trigger collection and extraction.

image

Testing:

When Collect ad hoc query using QDS : we have following output true coming from nuget
image

### CODE REVIEW

**Comment by Ramudaykumar** [comment]
QDS is available from 2016 and above? why this msg saying 2019? 

image

File: extensions/sql-migration/src/constants/strings.ts [hunk 1]
```diff
@@ -91,10 +91,12 @@ export const DATABASE_FOR_ASSESSMENT_PAGE_TITLE = localize('sql.migration.databa
...
+export const XEVENTS_ASSESSMENT_TITLE = localize('sql.migration.database.assessment.xevents.title', "Assess Ad-hoc or dynamic SQL");
+export const XEVENTS_ASSESSMENT_DESCRIPTION = localize('sql.migration.database.assessment.xevents.description', "For the selected databases, select the option you want to assess ad-hoc or dynamic SQL queries or any DML statements initiated through the application data layer. {0}");
 export const XEVENTS_ASSESSMENT_HELPLINK = localize('sql.migration.database.assessment.xevents.link', "Learn more");
 export const XEVENTS_ASSESSMENT_OPEN_FOLDER = localize('sql.migration.database.assessment.xevents.instructions', "Select a folder where extended events session files (.xel and .xem) are stored");
+export const QDS_ASSESSMENT_LABEL = localize('sql.migration.database.assessment.qds.label', "Using Query Data Store (this option is available for Microsoft SQL Server 2016 and later)");
+export const XEVENTS_LABEL = localize('sql.migration.database.assessment.xevents.label', "Using extended event session");
```
Comment (Ramudaykumar) [inline extensions/sql-migration/src/constants/strings.ts:99]: Extended event is not yet supported, this is for pre-exiting feature right?

**Comment by Ramudaykumar** [comment]
I didn't see any PR in sqltoolsservice to get latest SQL NuGet and Sqltoolsservice version upgrade here? How is this getting done?

File: extensions/sql-migration/src/constants/strings.ts (hunk 1 above)
Comment (stuti149) [reply extensions/sql-migration/src/constants/strings.ts:99]: yes, this is pre-existing one.

**Comment by stuti149** [comment]
> I didn't see any PR in sqltoolsservice to get latest SQL NuGet and Sqltoolsservice version upgrade here? How is this getting done?

@Ramudaykumar 

https://github.com/microsoft/sqltoolsservice/pull/2322

**Comment by stuti149** [comment]
> QDS is available from 2016 and above? why this msg saying 2019?
> 
> image

Screenshot is older one, strings are updated. Will update SS as well.

**Comment by Ramudaykumar** [comment]
> > I didn't see any PR in sqltoolsservice to get latest SQL NuGet and Sqltoolsservice version upgrade here? How is this getting done?
> 
> @Ramudaykumar
> 
> [microsoft/sqltoolsservice#2322](https://github.com/microsoft/sqltoolsservice/pull/2322)

Thanks, I have approved it, you will need to run official build of SQLToolsservie and update latest version in  extensions/sql-migration/package.json.

**Comment by Ramudaykumar** [comment]
> > QDS is available from 2016 and above? why this msg saying 2019?
> > image
> 
> Screenshot is older one, strings are updated. Will update SS as well.

Thanks, I can see in code. You may update the screenshot as it may confuse other reviewers as well.

**Review by Ramudaykumar — APPROVED** [review]

**Pull request merged**